streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractRocksDBSessionStoreTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals;
+
+public abstract class AbstractRocksDBSessionStoreTest extends AbstractSessionBytesStoreTest {
+
+    @Override
+    String getStoreName() {
+        return "rocksDB session store";
+    }
+
+    abstract StoreType storeType();
```

Comment:
we don't need extra abstract method, right?

Reply:
In fact, `AbstractRocksDBSessionStoreTest` is redundant, since its child can implement `getStoreType`. Also, `getStoreName` can be removed since `AbstractSessionBytesStoreTest` can pick up the name.

Reply:
`getStoreType` can be renamed to `storeType`

Reply:
@chia7712, Thanks for your comment, I think it' a good idea.